charts/linkerd2/values.yaml

```diff
@@ -92,7 +93,6 @@ webhookFailurePolicy: Ignore
 
 # controller configuration
 controllerImage: gcr.io/linkerd-io/controller
-controllerLogLevel: &controller_log_level info
```

Comment:
@alpeb Though we moved this flag to be global like we should, but we don't actually have a way to use this directly in `prometheus/values.yaml` as we can't template values there. and putting that in the templates directly means that we don't have a way to merge them and appending might cause prometheus to complain just like it did with other values in configmap (I will try this out today)